Kitodo/src/main/webapp/WEB-INF/templates/includes/head.xhtml

```diff
@@ -0,0 +1,26 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+                xmlns:h="http://xmlns.jcp.org/jsf/html">
+    <title><ui:insert name="title">Kitodo.Production</ui:insert></title>
+
+    <meta name="description" content="KITODO.PRODUCTION Base-Theme"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, shrink-to-fit=no"/>
+
+    <h:outputStylesheet name="css/kitodo.css"/>
+    <link href='http://fonts.googleapis.com/css?family=Raleway:300,400,500'/>
```

Comment:
I would suggest to switch to a protocol-less URI for loading this font as a secure connection would not allow to load a non-secure accessible data.

Reply:
Other question regarding to this font `Raleway`: it is allowed to use and distribute this font in our project?

Reply:
> Other question regarding to this font Raleway: it is allowed to use and distribute this font in our project?

According to https://www.fontsquirrel.com/license/raleway this font uses the [SIL Open Font License](https://en.wikipedia.org/wiki/SIL_Open_Font_License), which - according to Wikipedia - "_permits the fonts to be used, modified, and distributed freely (so long as the resulting fonts remain under the Open Font License). However, the copyright holder may declare the font's name as being a "Reserved Font Name", which modified versions then cannot bear. The License permits covered fonts to be freely embedded in documents under any terms, but it requires that fonts be packaged with software if they are sold._"

Reply:
Thank you for this explanation for the license of the font. Can you add a comment with a link to the license? Thank you.